pycaption/webvtt.py

```diff
@@ -77,31 +80,35 @@ def _parse(self, lines):
                 timing_line = i
                 last_start_time = captions[-1].start if captions else 0
                 try:
-                    caption = self._parse_timing_line(line, last_start_time)
+                    start, end, layout_info = self._parse_timing_line(
+                        line, last_start_time)
                 except CaptionReadError as e:
                     new_message = u'%s (line %d)' % (e.args[0], timing_line)
                     raise type(e), new_message, sys.exc_info()[2]
 
             elif u'' == line:
                 if found_timing:
-                    if caption.is_empty():
+                    if not nodes:
                         raise CaptionReadSyntaxError(
                             u'Cue without content. (line %d)' % timing_line)
                     else:
                         found_timing = False
+                        caption = Caption(
+                            start, end, nodes, layout_info=layout_info)
                         captions.append(caption)
-                        caption = None
+                        nodes = []
             else:
                 if found_timing:
-                    if not caption.is_empty():
-                        caption.nodes.append(CaptionNode.create_break())
-                    caption.nodes.append(CaptionNode.create_text(
+                    if nodes:
+                        nodes.append(CaptionNode.create_break())
+                    nodes.append(CaptionNode.create_text(
                         self._decode(line)))
                 else:
                     # it's a comment or some metadata; ignore it
                     pass
 
-        if caption and not caption.is_empty():
+        if all([start, end, nodes]):
```

Comment:
This seems like it wouldn't add the first caption if it starts at 0... EVEN IF it's a legitimate caption.